pkgs/tools/typesetting/tex/texlive/tlpdb.nix

```diff
@@ -39789,15 +41164,56 @@ texlive-scripts = {
   ];
   sha512.run = "3dbb2007ae7b80862265d6196f77a9d796f02f6914871f4dad094f8419040fe7064daf6ecf5dd611b5764dd4148d034a97d82aa22671199e8a3dc79373859c67";
   sha512.doc = "6976ba00c8ee50664aaa1f762231f297f01548a94d6a632b386845e7fa43b20b63342d58874e87869a73a9c23bba358f762f5cc3814690f870c6216679471a89";
+  hasManpages = true;
   hasRunfiles = true;
   hasTlpkg = true;
+  scriptExts = [
+    "lua"
+    "tcl"
+  ];
 };
 texlive-scripts-extra = {
   revision = 62517;
   sha512.run = "46ac37826d3c60de6c9260bf83d6275d49a35cbde88fb03481a050f92e87b698e9a94b2e520a74edc0417419f5a2dee53000a529b9c81ea6f6244a83480e56e7";
   sha512.doc = "22cf59bf4dafc7ad9425086bc0aaedb2bf5f7d8aa6ea9c65abde2d523be37665b9c9bee4acb399857eae03613e7241ca1d6099f43cab77a95c10eced8813ad80";
-  hasRunfiles = true;
-};
+  hasManpages = true;
+  hasRunfiles = true;
+};
+texlive-scripts-extra.binfiles = [
+  "allcm"
+  "allec"
+  "allneeded"
+  "dvi2fax"
+  "dvired"
+  "e2pall"
+  "kpsepath"
+  "kpsetool"
+  "kpsewhere"
+  "kpsexpand"
+  "mkocp"
+  "mkofm"
+  "ps2frag"
+  "pslatex"
+  "texconfig"
+  "texconfig-dialog"
+  "texconfig-sys"
+  "texlinks"
+];
+texlive-scripts.binfiles = [
+  "fmtutil"
+  "fmtutil-sys"
+  "fmtutil-user"
+  "man"
+  "mktexfmt"
+  "mktexmf"
+  "mktexpk"
+  "mktextfm"
+  "rungs"
```

Comment:
I just stumpled upon another bug this PR fixes. Currently gs specials (?) are broken in dvipdfmx (and thus also xetex) because dvipdfmx sifts them through ghostscript using the `rungs` wrapper script -- which we didn't package. Thus
```nix
dvipdfmx = runCommand "texlive-test-dvipdfmx" {
  nativeBuildInputs = [ (with texlive; combined.scheme-full) ];
  input = builtins.toFile "dvipdfmx-sample.tex" ''
    \documentclass[12pt]{article}
    \usepackage[dvipdfmx]{graphics}
    \begin{document}
    \includegraphics{example-image-a.eps}
    \end{document}
  '';
} ''
  cp $input document.tex

  latex document.tex
  dvipdfmx -vvvv document.dvi

  mkdir "$out"
  mv document* "$out"/
'';
```
failed with something like
```
dvipdfmx:warning: Filtering file via command -->rungs -q -dSAFER -dNOPAUSE -dBATCH -dEPSCrop -sPAPERSIZE=a0 -sDEVICE=pdfwrite -dCompatibilityLevel=1.5 -dAutoFilterGrayImages=false -dGrayImageFilter=/FlateEncode -dAutoFilterColorImages=false -dColorImageFilter=/FlateEncode -dAutoRotatePages=/None -sOutputFile='/build/dvipdfm-x.0414c522dac65f0e190d029fd531f8ec' '/nix/store/vyv2s97b0kzb3cp153sfija9l21y3g85-texlive-combined-full-2022-final-texmfdist/tex/latex/mwe/example-image-a.eps' -c quit<-- failed.
dvipdfmx:warning: Image format conversion for "/nix/store/vyv2s97b0kzb3cp153sfija9l21y3g85-texlive-combined-full-2022-final-texmfdist/tex/latex/mwe/example-image-a.eps" failed...
```

I'm kind of shocked that this went unnoticed for so long. Great that this is fixed!

Reply:
Ah, so `rungs` is quite important after all. I hastily decided not to patch/wrap `rungs` with Ghostscript because it would increase the closure size of collection-basic, so virtually every scheme.

Ok, so new small task, grep the scripts for `rungs` and see how hard it would be to wrap those only...

Reply:
It turns out that tex4ht is the only other user of `rungs`. I decided to add Ghostscript when the package dvipdfmx is present, but ignored tex4ht since it's not a core dependency (an entirely subjective statement!).